db/schema.rb

```diff
@@ -146,10 +146,12 @@
     t.string "slug", limit: 10000, null: false
     t.string "url"
     t.string "description", default: "", null: false
+    t.string "tags_links_digest"
```

Comment:
Once this is populated with data should it be null: false?

Reply:
I'm not sure actually. If we have a not null constraint, we'd have to put in the hash of '{}' for every row of the table, but would we gain much from that?